packages/eslint-config-widen/package.json

```diff
@@ -1,15 +1,11 @@
 {
   "author": "Widen",
   "dependencies": {
-    "eslint-config-prettier": "^9.0.0"
+    "eslint-config-prettier": "^9.1.0"
   },
   "description": "Widen's shared ESLint config.",
   "exports": {
-    ".": "./lib/index.js",
-    "./jest": "./lib/jest.js",
```

Comment:
No longer needed?

Reply:
Just changed the way we export: import { base, typescript, react, playwright, jest } from 'eslint-config-widen'